packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -25,12 +25,14 @@ export interface MenuItemProps<T = any> {
   active?: boolean;
 
   tabIndex?: number;
+
+  children?: ReactNode;
```

Comment:
allowing _anything_ here feels a bit wrong since we're only ever expecting these children to be another menu. what about having a separate prop, something like `childItems`?